dashboard/app/mail_bug.txt

```diff
@@ -15,7 +15,7 @@ git tree:       {{.KernelRepoAlias}}
 {{end}}{{if .ReproCLink}}C reproducer:   {{.ReproCLink}}
 {{end}}{{if and .Moderation .Maintainers}}CC:             {{.Maintainers}}
 {{end}}{{if and (not .NoRepro) (not .ReproCLink) (not .ReproSyzLink)}}
-Unfortunately, I don't have any reproducer for this crash yet.
+Unfortunately, I don't have any reproducer for this issue yet.
 {{end}}
 {{if .BisectCause}}{{if .BisectCause.Commit}}The bug was bisected to:
```

Comment:
We also have a number of uses of the word "bug" here and in other emails. Does it look good to you to replace all of these with "issue"?